data/helpers.d/nodejs

```diff
@@ -126,6 +127,9 @@ ynh_install_nodejs () {
 
     # If n is not previously setup, install it
     if ! test $(n --version > /dev/null 2>&1)
```

Comment:
```suggestion
    if ! $n_install_dir/bin/n --version > /dev/null 2>&1
```

After some tests, i found that the current `test` doesn't work. n is reinstalled each time.
Can someone else test this helper please?

Reply:
@kay0u I have tested your code with this configurations:
- install `n` when `n` is not installed
- install `n` when `n`< 6.5.1
- do nothing when `n` already installed

It seems to work...